src/lingrex/trimming.py

```diff
@@ -0,0 +1,201 @@
+"""
+Trimming functionalities in lingrex.
+"""
+import random
+import collections
+from lingpy.sequence.sound_classes import token2class
+
+
+def revert(alms):
```

Comment:
I'd also tend to rename this function to `get_sites` or `iter_sites` or similar.

Considering that several aspects of alignment sites are computed below (gap ratio, sound class, etc.), it may be useful to actually introduce a `Site` class:
```python
class Site:
    def __init__(self, aligned_sounds):
        self.sounds = aligned_sounds
        self.gap_ratio = ...
        self.soundclass = ...
```
and then have the trimming functions accept a list of `Site`s?

Reply:
`get_skeleton` and `get_profile` would then be replaced with
```python
[site.gap_ratio for site in sites]
...
```
or access to the ratio/soundclass could be by index: `sites[idx].gap_ratio`.